Chummer/Classes/clsCharacter.cs

```diff
@@ -10898,7 +10898,7 @@ public int CurrentCounterspellingDice
             }
         }
         #region Dodge
-        public int Dodge => REA.TotalValue + INT.TotalValue + TotalBonusDodgeRating;
+        public int Dodge => REA.TotalValue + INT.TotalValue + TotalBonusDodgeRating + WoundModifier;
```

Comment:
Could you update the dependency graph as well so that `Dodge` gets its `PropertyChanged` fired whenever `WoundModifier` changes?

Reply:
Can do, but it already worked like this without any problems, for some reason and then forgot about those.